### ISSUE TITLE
If prowler doesn't have permissions to run a check (e.g. in cross-account setups) it prints `FAIL`. Should print `WARN` instead.

### ISSUE THREAD

**Issue by nicdoye** [prowler-cloud/prowler#380]
Running Check 2.6 in a setup where the CloudTrail S3 bucket is in a different account.

```
 2.6  [check26] Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket (Scored)
       FAIL! access logging is not enabled in aws-controltower-logs-XXXXXXXXXXXX-us-east-2 CloudTrail S3 bucket!
```

The bucket does have access logging enabled, but, because the bucket is in a different account, we don't have permissions to find this out.

From an Account Factory account (YYYYYYYYYYYY) where I ran prowler:
```
➜ aws s3api get-bucket-logging --bucket aws-controltower-logs-XXXXXXXXXXXX-us-east-2

An error occurred (AccessDenied) when calling the GetBucketLogging operation: Access Denied

➜ echo $?
255
```

From Control Tower Log Archive account (XXXXXXXXXXXX):
```
➜ aws s3api get-bucket-logging --bucket aws-controltower-logs-XXXXXXXXXXXX-us-east-2
{
    "LoggingEnabled": {
        "TargetBucket": "aws-controltower-s3-access-logs-XXXXXXXXXXXX-us-east-2",
        "TargetPrefix": ""
    }
}

➜ echo $?
0
```

**Comment by toniblyx** [comment]
I found that check23 may have the same problem with cross accounts configurations like with Control Tower

**Comment by toniblyx** [comment]
this was fixed on PR #384
